Mathlib/CategoryTheory/Galois/Basic.lean

```diff
@@ -202,14 +202,37 @@ noncomputable def fiberEqualizerEquiv {X Y : C} (f g : X ⟶ Y) :
   · exact Types.equalizerIso (F.map f) (F.map g)
 
 /-- The fiber of the pullback is the fiber product of the fibers. -/
-@[simp]
 noncomputable def fiberPullbackEquiv {X A B : C} (f : A ⟶ X) (g : B ⟶ X) :
```

Comment:
I am ok for removing the `simp` attribute, but then I believe that you should add `simp` lemmas similar to `fiberBinaryProductEquiv_symm_fst_apply`.